ocaml/idl/datamodel_repository.ml

```diff
@@ -73,6 +82,37 @@ let introduce =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_bundle =
+  call ~name:"introduce_bundle" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Add the configuration for a new bundle repository"
+    ~versioned_params:
```

Comment:
I understand that a new API call's parameters are versioned together with the call/message, instead of versioned explicitly.

Reply:
You could use `~params` rather than `~versioned_params` in this case.